rootfs-builder/README.md

```diff
@@ -1,14 +1,15 @@
 * [Supported base OSs](#supported-base-oss)
+* [Rootfs requirements](#rootfs-requirements)
 * [Creating a rootfs](#creating-a-rootfs)
 * [Creating a rootfs with kernel modules](#creating-a-rootfs-with-kernel-modules)
-* [Build a rootfs using Docker*](#build-a-rootfs-using-docker)
+* [Build a rootfs using Docker](#build-a-rootfs-using-docker)
 * [Adding support for a new guest OS](#adding-support-for-a-new-guest-os)
     * [Create template files](#create-template-files)
     * [Modify template files](#modify-template-files)
     * [Expected rootfs directory content](#expected-rootfs-directory-content)
-    * [Optional - Customise the rootfs](#optional---customise-the-rootfs)
+    * [Optional - Customise the rootfs](#optional-customise-the-rootfs)
```

Comment:
Can we check here - I think the tag does actually have `---` in its name... (link failed in my 'view file' view)

Reply:
Fixed.